### DISCUSSION TITLE
Learning Group: Machine Learning in Python

### DISCUSSION THREAD

**Discussion by kraus-s** [network-digital-old-norse-studies/network-digital-old-norse-studies.github.io#26] [Ideas]
Hi all

at the last Basel Group meeting we were discussing Machine Learning with Python and how to get started. We came up with the idea that we could learn together and start a little learning group where we watch [YouTube tutorials by W. J. B. Mattingly](https://www.youtube.com/pythontutorialsfordigitalhumanities) in preparation for short group sessions. In the group sessions we can share code, try and answer questions together and help each other with our projects.
If you want to join comment below and we will use this thread to organise the learning group.

Cheers!
Sven

**Comment by BalduinLandolt** [comment]
I'm happy to join, if time (which is a rare resource, these days) allows. Did you have any specific videos in mind that would make for a good starting point?

**Comment by Arya125-droid** [comment]
I think Supervised learning from coursers is the best to start with in Machine Learning

**Comment by ElineElmiger** [comment]
I'd also like to join!

**Comment by ElineElmiger** [comment]
Do you have a good beginner-friendly video or article that explains what is possible with machine learning in python for beginners? I have some ideas for small projects but I have no idea whether they would be good for this kind of learning group

**Comment by kraus-s** [comment]
Hey everyone,
let's get the ball rolling! 
My suggestion would be to start with this playlist here: https://www.youtube.com/playlist?list=PL2VXyKi-KpYuPKEdLpNRuXSjTzqNfyG-N
and meet up every other week for anywhere between 30 minutes and one hour to discuss. Say Tuesdays at 10?
And I am not sure, but did you want to participate, @MaditaVerena ?

**Reply by ElineElmiger** [reply]
Tuesdays at 10 works for me, except for next week and the one after that (19.3. & 26.3.)

**Reply by kraus-s** [reply]
Then lets make April 2nd our first meeting!

**Reply by BalduinLandolt** [reply]
I'm afraid Tuesdays don't work for me. =(  
But have fun and good luck. And keep me posted how it's going!
